src/main/java/TaskManager.java

```diff
@@ -0,0 +1,126 @@
+import java.util.ArrayList;
+
+public class TaskManager {
+
+    protected ArrayList<Task> tasks;
```

Comment:
I like how you use a resizable array rather than just a fixed-sized array to store the tasks.